README.md

```diff
@@ -1,2 +1,20 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) the task is checking to see whether or not any elements are duplicated
+b) as they are both nested lists checking one element in list i compared to every element in list j if there was only one duplicate at the end of i with the end of j would be worst case scenario 
```

Comment:
Explanation could be clearer, but I think you get the idea that the nested loops lead to the O(n^2) complexity